### ISSUE TITLE
Random high CPU utilization on 32 logical CPUs machine

### ISSUE THREAD

**Issue by top-5** [catalincd/resource-monitor#4]
Hello,

Thank you for the wonderful project! I'm running on AMD Ryzen 9 with 16-core, 32 logical processors. I hit an issue that when the number of cores is 32 - the code actually starts reporting junk random value for the last core. This distorts the view: it'd appear that the utilization on one last core is in some thousands instead of zero.

You could either increase the limit to 33 or 65 or something like that, - that'd fix the issue..

Basically `#define MAX_CPUS 33` would fix it.

I wonder if you accept PRs, I can send in a fix.

Thank you. 🙇‍♂️
